io_mesh_w3d/common/structs/rgba.py

```diff
@@ -62,10 +62,10 @@ def write_f(self, io_stream):
     def to_vector_rgba(self, alpha=None, scale=255.0):
         if alpha is None:
             alpha = self.a / scale
-        return (self.r / scale, self.g / scale, self.b / scale, alpha)
+        return self.r / scale, self.g / scale, self.b / scale, alpha
```

Comment:
You could also create a tuple and divide that by scale, instead of dividing each scalar individually

Reply:
nope^^
![image](https://user-images.githubusercontent.com/10086882/74328530-61385d00-4d8e-11ea-87dc-0b80567a8570.png)